scaling_performance/install_practices.adoc

```diff
@@ -0,0 +1,80 @@
+[[scaling-performance-install-best-practices]]
+= Recommended Installation Practices
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+
+[[scaling-performance-preinstalling-dependencies]]
+== Pre-installing Dependencies
+
+Save time by ensuring that the RPMs for `atomic-openshift-*`, `iptables`,
+`docker`, and other required packages have been set up before installing
+{product-title}. 
+
+If the dependencies have not been pre-installed, a node host will access the
```

Comment:
Done. And I can link this out to the "gold image" section, after renaming it as you say in the email. Cool cool.